README.md

```diff
@@ -50,20 +50,26 @@ assert_equal "🎉", result
 verify { glass.pour!(:a_drink) }
 ```
 
-# The chaser
-
-And that's not all! Mocktail can do a whole lot more:
-
-* Singleton methods on classes and modules can be replaced with
-  [Mocktail.replace(type)](#mocktailreplace) while still preserving thread
-  safety
-* Arity of arguments and keyword arguments of faked methods is enforced—no more
-  tests that keep passing after an API changes
-* Super-duper detailed error messages when verifications fail
-* Dynamic stubbings based on the arguments passed to the actual call
-* Built-in [argument matchers](#mocktailmatchers) and a simple API for adding
-  [custom matchers](#custom-matchers)
-* [Argument captors](#mocktailcaptor) for assertions of very complex arguments
+## Why order?
+
+Besides a lack of hangover, Mocktail offers several advantages over other
+mocking libraries:
+
+* **Fewer hoops to jump through**: [`Mocktail.of_next(type)`] avoids the need
```

Comment:
FWIW I'm glad you're putting this one front and center. It was the first big feature/benefit combo that smacked me in the face when perusing the docs. 💯

Reply:
Yeah if nothing else changed, I think putting this here is the thing I'd still like to see happen.

Reply:
Yeah if nothing else changed, I think putting this here is the thing I'd still like to see happen.